pytorch_lightning/accelerators/accelerator.py

```diff
@@ -274,27 +274,23 @@ def validation_step_end(self, output: Optional[STEP_OUTPUT]) -> Optional[STEP_OU
     def backward(
         self,
         closure_loss: Tensor,
-        optimizer: Optimizer,
-        optimizer_idx: int,
-        should_accumulate: bool,
         *args: Any,
         **kwargs: Any,
     ) -> Tensor:
         """Forwards backward-calls to the precision plugin.
 
         Args:
             closure_loss: a tensor holding the loss value to backpropagate
-            should_accumulate: whether to accumulate gradients
         """
-        self.training_type_plugin.pre_backward(closure_loss, should_accumulate, optimizer, optimizer_idx)
+        self.training_type_plugin.pre_backward(closure_loss)
+        closure_loss = self.precision_plugin.pre_backward(self.lightning_module, closure_loss)
 
-        output = self.precision_plugin.backward(
-            self.lightning_module, closure_loss, optimizer, optimizer_idx, should_accumulate, *args, **kwargs
-        )
+        self.precision_plugin.backward(self.lightning_module, closure_loss, *args, **kwargs)
```

Comment:
what about the other args here? as `optimizer`, `optimizer_idx` and `should_accumulate`? This is a BC right?

Reply:
Optimizer and opt index are passed with args kwargs

Should accumulate wasnt passed to the LightningModule hook so can be removed as we no longer use it

Reply:
Hi @carmocca, I noticed that in some cases the optimizer is being directly plumbed down to `loss.backward(*args, **kwargs)` through `args`, which is breaking some of our tests (bisection pointed to this PR). It'd take some time to come up with a repro since the setup is a bit complicated. Asking first in case you know what could have gone wrong.

cc: @ananthsub, @kandluis

Reply:
Perhaps you are using `manual_backward(loss, optimizer)`?

The optimizer argument was removed (in 1.4,  after the deprecation process) and if you are still passing it will be treated as part of the `*args` here

Reply:
@carmocca thanks for the prompt response! Is not passing optimizer to manaul_backward the recommended fix here?

Reply:
Correct. The argument was removed in #8287 and was unused since 1.2